script/Deploy.s.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.17;
+
+import {Script} from "forge-std/Script.sol";
+import {GitcoinGovernor} from "src/GitcoinGovernor.sol";
+
+contract DeployInput {
+  uint256 constant INITIAL_VOTING_DELAY = 13_140;
+  uint256 constant INITIAL_VOTING_PERIOD = 40_320;
+  uint256 constant INITIAL_PROPOSAL_THRESHOLD = 1_000_000e18;
```

Comment:
Nit, maybe add comment with what these values translate to in hours or days / where we got them from / if these are prod values or testing values that might change. Feel free to resolve this comment yourself

Reply:
For the moment these values are just pulled from the existing Governor. We'll like update them with feedback from Gitcoin at some point. Some comment documentation at that point makes sense 👍